gamemodes/terrortown/gamemode/client/cl_changes.lua

```diff
@@ -678,6 +678,22 @@ local function MakePanel(panel, change)
 	panel.htmlSheet = html
 end
 
+
+function GetSortedChanges()
+	CreateChanges()
+
+	-- sort changes list by date
+	table.sort(changes, function(a, b)
```

Comment:
You should introduce caching here. Sorting is a performance heavy task and should not get executed every time, the changelog opens.

Reply:
Probably. It has been this way eversince. The changelog gets reworked anyway